packages/app/src/themes/colorTypes.tsx

```diff
@@ -73,44 +154,52 @@ export const gold = {
     A700: vadsGold.A700,
 };
 
-const vadsGray = {
-    50: '#f1f1f1',  
-    100: '#f1f1f1', //$color-gray-lightest
-    200: '#d6d7d9', //$color-gray-lighter 
-    300: '#e4e2e0', //$color-gray-warm-light
-    400: '#dce4ef', //$color-gray-cool-light
-    500: '#aeb0b5', //$color-gray-light
-    600: '#757575', //$color-gray-medium
-    700: '#E8A937', //$color-gray
-    800: '#494440', //$color-gray-warm-dark
-    900: '#323a45', //$color-gray-dark
-    A100: '#323a45',
-    A200: '#323a45',
-    A400: '#323a45',
-    A700: '#323a45'
+/**
+ * VADS Green
+ */
+const vadsGreen = {
+    50: '#FCFAFA',
+    100: '#F1F1F1', //$color-green-lightest
+    200: '#E4E2E0',
+    300: '#D6D7D9', //$color-green-lighter
+    400: '#7AA688',
+    500: '#AEB0B5', //$color-green-light
+    600: '#757575', //$color-green
+    700: '#5B616B',
+    800: '#494440', //$color-green-darker
+    900: '#323A45',
+    A100: '#F1F1F1',
+    A200: '#DCDCDB',
+    A400: '#CCCCCC',
+    A700: '#535559',
 };
 
-export const gray = {
-    50: vadsGray[50],
-    100: vadsGray[100],
-    200: vadsGray[200],
-    300: vadsGray[300],
-    400: vadsGray[400],
-    500: vadsGray[500],
-    600: vadsGray[600],
-    700: vadsGray[700],
-    800: vadsGray[800],
-    900: vadsGray[900],
-    A100: vadsGray.A100,
-    A200: vadsGray.A200,
-    A400: vadsGray.A400,
-    A700: vadsGray.A700,
+export const green = {
+    50: vadsGreen[50],
+    100: vadsGreen[100],
+    200: vadsGreen[200],
+    300: vadsGreen[300],
+    400: vadsGreen[400],
+    500: vadsGreen[500],
+    600: vadsGreen[600],
+    700: vadsGreen[700],
+    800: vadsGreen[800],
+    900: vadsGreen[900],
+    A100: vadsGreen.A100,
+    A200: vadsGreen.A200,
+    A400: vadsGreen.A400,
+    A700: vadsGreen.A700,
 };
 
+
 /**
- * Variable for common colors like white/black
+ * VADS Misc for comon colors
```

Comment:
```suggestion
 * VADS Misc for common colors
```
Nitpick on spelling.